### ISSUE TITLE
Missing code from first few code cells?

### ISSUE THREAD

**Issue by jirvingphd** [learn-co-curriculum/dsc-1-02-17-implementing-statistics-with-functions-lab#8]
I'm wondering if somehow this file was accidentally edited? (2 examples) 
Please see attached annotated screenshot .

1. If you look at the notebook, the second cell asks for our thoughts about the histogram, but there is no histogram (the section below has us create the histogram).

2. Below that, it says it is now saving the height column as a variable, but there is a just a random letter v and print commands in that cell. 
![error image](https://user-images.githubusercontent.com/44880739/52720178-24683900-2f75-11e9-8900-1b1342d8516d.jpg)
Thanks!

**Comment by gdurante2019** [comment]
Yes--I saw this too.  (I also posted in AAQ, since I'd like for people to use that.)

Flatiron staff:  please check this out and fix.  Thank you!
